src/src/WebServer/NotificationPage.cpp

```diff
@@ -142,7 +142,19 @@ void handle_notifications() {
         html_TD();
         addHtmlInt(NotificationSettings.Port);
       }
-      else {
+      else
+      {
+          //MFD: we display the GPIO
+          if (NotificationSettings.Pin1>=0){
+            addHtml(F("GPIO-"));
+            addHtmlInt(NotificationSettings.Pin1);
+          }
+          if (NotificationSettings.Pin2>=0)
+          {
+            html_BR();
+            addHtml(F("GPIO-"));
+            addHtmlInt(NotificationSettings.Pin2);
+          }
```

Comment:
I was just reading through this PR code and just realized I had (a long time ago) made functions to generate these descriptions.

So I searched through the code and I think the best function to use here is `GpioToHtml` as it is also used on the DevicesPage to format generic pins.

Reply:
that method is private to the DevicesPage we will have to move it in the html helper file then I guess I can use it.

Reply:
done and rename it to addGpioHtml to match the rest of the methods